src/app/modules/main/wallet_section/send/module.nim

```diff
@@ -186,4 +192,16 @@ method suggestedRoutesReady*(self: Module, suggestedRoutes: string) =
   self.view.suggestedRoutesReady(suggestedRoutes)
 
 method getEstimatedTime*(self: Module, chainId: int, maxFeePerGas: string): int = 
-  return self.controller.getEstimatedTime(chainId, maxFeePerGas).int
\ No newline at end of file
+  return self.controller.getEstimatedTime(chainId, maxFeePerGas).int
+
+method switchAccount*(self: Module, accountIndex: int) =
+  var walletAccount = self.controller.getWalletAccountByIndex(accountIndex)
+  if not walletAccount.isNil:
+    self.view.switchSenderAccountByAddress(walletAccount.address)
+    self.view.switchReceiveAccount(accountIndex)
```

Comment:
what happens in case of not nil? shall we select account index 0?

Reply:
wouldnt change, default is 0 and otherwise it will stay as the last user set account in the modal